src/app/Rules/Rules.tsx

```diff
@@ -0,0 +1,197 @@
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import * as React from 'react';
+import { Card, CardBody, EmptyState, EmptyStateIcon, Title } from '@patternfly/react-core';
+import { SearchIcon } from '@patternfly/react-icons';
+import { Table, TableBody, TableHeader, TableVariant, ICell, ISortBy, info, sortable } from '@patternfly/react-table';
+import { ServiceContext } from '@app/Shared/Services/Services';
+import { NotificationCategory } from '@app/Shared/Services/NotificationChannel.service';
+import { useSubscriptions } from '@app/utils/useSubscriptions';
+import { BreadcrumbPage } from '@app/BreadcrumbPage/BreadcrumbPage';
+import { LoadingView } from '@app/LoadingView/LoadingView';
+
+interface Rule {
+  name: string;
+  description: string;
+  matchExpression: string;
+  archivalPeriodSeconds: number;
+  preservedArchives: number;
+  maxAgeSeconds: number;
+  maxSizeBytes: number;
+}
+
+export const Rules = () => {
+  const context = React.useContext(ServiceContext);
+  const addSubscription = useSubscriptions();
+
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [sortBy, setSortBy] = React.useState({} as ISortBy);
+  const [rules, setRules] = React.useState([] as Rule[]);
+
+  const tableColumns = [
+    {
+      title: 'Name',
+      transforms: [ sortable ],
+    },
+    { title: 'Description', },
+    {
+      title: 'Match Expression',
+      transforms: [
+        info({
+          tooltip: 'A code-snippet expression which must evaluate to a boolean when applied to a given target. If the expression evaluates to true then the rule applies to that target.'
+        })
+      ],
+    },
+    {
+      title: 'Archival Period',
+      transforms: [
+        info({
+          tooltip: 'Period in seconds. Cryostat will connect to matching targets at this interval and copy the relevant recording data into its archives.'
+        })
+      ],
+    },
+    {
+      title: 'Preserved Archives',
+      transforms: [
+        info({
+          tooltip: 'The number of recording copies to be maintained in the Cryostat archives. Cryostat will continue retrieving further archived copies and trimming the oldest copies from the archive to maintain this limit.'
+        })
+      ],
+    },
+    {
+      title: 'Maximum Age',
+      transforms: [
+        info({
+          tooltip: 'The maximum age in seconds for data kept in the JFR recordings started by this rule.'
+        })
+      ],
+    },
+    {
+      title: 'Maximum Size',
+      transforms: [
+        info({
+          tooltip: 'The maximum size in bytes for JFR recordings started by this rule.'
+        })
```

Comment:
Maybe you could indicate here that `Max Age: 0` and `Max Size: -1` correspond to unlimited age/size, to make it extra clear that `0` and `-1` are special values